### ISSUE TITLE
Implement temperature processor

### ISSUE THREAD

**Issue by qualiteger** [qualiteg/ChatStream#46]
基本　logits = logits / temperature　のみでOK

    ・温度が1より大きい場合（例: temperature = 1.5など）:

        出力の確率分布が「フラット」になる
        これは、低確率の出力も選択される可能性が増えるので、結果として、モデルの出力の多様性が増す
        一方で元の確率分布から逸脱する可能性も高まる

    ・温度が1より小さい場合（例: temperature = 0.5など）:
        出力の確率分布が「ピーク」を持ちやすくなる
        高確率の出力がさらに選択される可能性が増えることを意味するので、
    　　　結果として、モデルの出力は元の確率分布に従いやすくなり、
        予測がより確実になるが、多様性は失われる可能性がある

    ・温度が非常に低い場合（例: temperatureが1e-4未満）:
    　   最も確率の高い出力だけが選択されるようになる

**Comment by qualiteger** [comment]
#42 のサブタスク
